README.md

```diff
@@ -74,7 +80,7 @@ import chromatism from 'chromatism';
 ## Functions
 All functions can take any colour type. **You can even mix colour types** if a function takes more than one. [A list of colour types are available here](#colour-modes). Return values are also not bound by the function. Any functions that return colours, return a colour object, which contains getters for each colour type. If you take the return value of one of these functions and get `returnValue.hex` for example, you will get the hexadecimal equivalent value.
 
-You can also chain these functions. The last parameter will be piped in from the output of the parent function.
+In previous versions (Pre. v2.6), you could chain functions. This functionality has been removed to save space on the bundle.
```

Comment:
Instead of `(Pre. v2.6)` it might be better to just say `(2.x)` - there's nothing stopping us from going back and pushing patch versions of version 2, and the semver syntax seems a bit clearer anyway.